admin/sql/create_tables.sql

```diff
@@ -42,4 +42,45 @@ CREATE TABLE api_compat.session (
     ts        TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 );
 ALTER TABLE api_compat.session ADD CONSTRAINT session_sid_uniq UNIQUE (sid);
+
+CREATE TABLE statistics.user (
+    user_id                 INTEGER NOT NULL, -- PK and FK to "user".id
+    artists                 JSONB,
```

Comment:
OK, this seems fine. I wasn't sure if it would be a better idea to split this data into separate fields.

It sounds like you're only ever going to write this field in bulk - that is, compute it all at once and then write it to the field, replacing whatever was there. Same for reading - you're always going to want to get all of the data at once in order to display it. If this is the case, then I'm fine with this way of storing the data.